src/Pages/ResultsFixtures/View.elm

```diff
@@ -77,6 +77,21 @@ dayResultsFixtures responsive leagueGamesForDay =
         ]
         (List.map (gameRow responsive) leagueGamesForDay.games)
 
+formatPlayersCount : List (String, Int) -> List String
+formatPlayersCount =
+            List.map
```

Comment:
I would probably move `List.map` in to `gameRowScorers`.

It is best to make your functions work with primitives where possible, and then compose them (by using `List.map` and suchlike).

If this is done, the function becomes more reusable, and you can then use it with `Maybe.map` and `Result.map` and so on, as well as just with a list. I know its not a big deal, and it probably isn't going to be reused anywhere for a while, but its a good habit to get in to.